src/views/accounts/AccountChangeSecret/ExecutionDetail/AccountChangeSecretRecord.vue

```diff
@@ -150,10 +150,14 @@ export default {
                   label: this.$t('Failed')
                 },
                 {
-                  label: this.$t('Execution'),
-                  value: 'execution_id'
+                  value: 'pending',
+                  label: this.$t('Pending')
                 }
               ]
+            },
+            {
+              label: this.$t('ExecutionID'),
+              value: 'execution_id'
             }
           ]
         },
```

Comment:
The main difference between the provided options is that the 'label' key is not listed in one of them. Additionally, a new option with the label "Status" has been added to one of the options lists. No obvious issues or irregularities are found in these changes. To optimize this code further, it could be useful to include more specific instructions on each item's usage based on its values (e.g., execution IDs versus pending states) for better understanding and maintainability.